index.js

```diff
@@ -129,6 +129,7 @@ class TuyaDevice extends EventEmitter {
       devId: this.device.id,
       t: Math.round(new Date().getTime() / 1000).toString(),
       dps: {},
+      cid: options.cid,
```

Comment:
This needs to get added to the above documentation.

Reply:
Done! Although I now changed cid attribute to be a tuya device attribute instead of a get/set function attribute in the latest commit :)

Reply:
I could be wrong on this but would it make more sense to have it passed in the function parameters like you had before? Don't you need multiple instances of TuyAPI now to talk to a single gateway if there's multiple subdevices connected to it?

Reply:
I guess it depends how we want to look at it from an API perspective: 
- 1 device (gateway) controlling multiple subdevices by cid
- or multiple devices (e.g lock, heating) controlled through the same ip & id

what do you prefer?

Reply:
I would say the second, but thinking through this some more I don't think this would work in practice since Tuya devices tend to be flaky if more than one TCP connection is open to them at a time. So let's go with the first for now (include `cid` in get / set options rather than constructor).

Reply:
Done! :)